### PR TITLE
Moar beautiful

### PR DESCRIPTION
**Pull request by emilyville**
Branch for new patient nav, still working out some kinks. Will generate list of issues that are relevant, but might as well move towards merging for now.

looks like this!
screen shot 2015-07-07 at 3 54 45 pm

Issues (will create soon w/ relevant mockups):
- submit button missing from patient edit page
- editing patient removes patient nav
- all sidebar content is still static and has not yet incorporated new stuff from Craig
- many fields in top nav don't have the correct variables and are currently static.

### CODE REVIEW

**Comment by awong-dev** [comment]
@emilyville BTW, do you mind squashing down some of the local commits? Trying to reduce unnecessary revision history churn on things like the Gemfile.

**Comment by awong-dev** [comment]
Is this ready for commit?

**Comment by emilyville** [comment]
Ok @awong-dev methinks ready for commit, pending your :+1: !

I saw your note about squashing local commits. I don't know what you mean so let's follow up later.

File: app/assets/stylesheets/sci.css.scss [hunk 1]
```diff
@@ -2,19 +2,111 @@
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
+.navbar-inverse#sci-nav {
+  background-color: #1D3D46;
+  border-color: #12252B;
+  border-radius: 0px;
+
+// .navbar-inverse .navbar-nav > .active > a, .navbar-inverse .navbar-nav > .active > a:hover, .navbar-inverse .navbar-nav > .active > a:focus
```
Comment (awong-dev) [inline app/assets/stylesheets/sci.css.scss]: remove comment?

File: app/assets/stylesheets/sci.css.scss [hunk 2]
```diff
@@ -2,19 +2,111 @@
...
   border-radius: 0;
   margin-top: -20px;
   border-top: 1px solid #999;
-  border-bottom: 1px solid #999;
+  min-height: 90px;
+  border-bottom: 1px solid #ddd;
+
+  h4 {
+    color: #1D3D46;
+    }
```
Comment (awong-dev) [inline app/assets/stylesheets/sci.css.scss]: nit: indentation looks off

**Pull request merged**